projects/plugins/protect/src/js/components/diff-viewer/styles.module.scss

```diff
@@ -0,0 +1,52 @@
+.diff-viewer {
+	font-size: var( --font-body );
+	line-height: 1.5;
+}
+
+.diff-viewer__filename {
+	padding: 4px 8px;
+	background-color: var( --jp-gray-10 );
+	font-weight: 600;
```

Comment:
I think we should we add overflow-x: scroll here in case the file path is too big.